content/strategy-goals/strategy/code-graph/code-insights/index.md

```diff
@@ -123,31 +123,42 @@ Users do not need to learn a new query language or format to use Code Insights.
 
 - Code Insights won't avoid introducing net new features if no other Sourcegraph feature has yet built them
 
-## Where we are now
+### Code Insights is a product _with graphs_, not a _graphing product_
+
+The primary value of Code Insights comes from the questions it can answer for its users in service of [Sourcegraph use cases](../../index.md#use-cases). While Code Insights will continue to add graph UI features as customers demonstrate need and value –- or even just desire -- we will primarily focus on building new insight types or graph flows (such as monitoring, cross-feature integrations, and filtering/drilldown ability) rather than new chart types or especially fancy charting customizations.
+
+**What this looks like:**
 
-Code Insights is a [Beta Feature](../../../../departments/product-engineering/product/process/beta_and_experimental_feature_labels.md). It entered Beta in August 2021, and will launch to General Availability in FY23Q1.
+- Even in rarer cases when we don't have user feedback to differentiate the priority of two features, we'll prioritize features that let you answer more types of questions, or answer them more specifically, rather than features that let you change the graph display for aesthetic purposes
+- We sometimes delay "standard" graphing UI features, such as persisted re-orderable dashboard displays, because they won't best unlock value for our users relative to other work
 
-This means that the Code Insights team is singularly focused on bringing Code Insights into general availability. To reach that goal, Code Insights will be building stability, features, and core product experiences based on the feedback we get from our beta participants.
+**What this does not look like:**
+
+- Code Insights will still prioritize adding new chart types if they _do_ enable insights to answer new questions, such as a heatmap chart type to visualize changes in a non-Cartesian coordinates way.
+
+## Where we are now
 
-After the Beta, Code Insights will be a standalone feature with its own pricing. The pricing and packaging details are [internally available](https://docs.google.com/document/d/11Y5ZDIT_nCwkobGzVgseM7vgmk5Hkt-4UZHvivHwN7A/edit#heading=h.tivmnwrtt3gg).
+Code Insights is generally available and a paid add-on feature. The pricing and packaging details are [internally available](https://docs.google.com/document/d/11Y5ZDIT_nCwkobGzVgseM7vgmk5Hkt-4UZHvivHwN7A/edit#heading=h.tivmnwrtt3gg).
```

Comment:
Maybe also link to the publicly available content here too (https://about.sourcegraph.com/code-insights/) since you can have readers who might be interested in where you are now but don't have access to the internal pricing and packaging info.

Reply:
```suggestion
[Code Insights is generally available](https://about.sourcegraph.com/code-insights/) and a paid add-on feature. The pricing and packaging details are [internally available](https://docs.google.com/document/d/11Y5ZDIT_nCwkobGzVgseM7vgmk5Hkt-4UZHvivHwN7A/edit#heading=h.tivmnwrtt3gg).
```